src/content/docs/new-relic-control/agent-control/architecture.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Understanding Agent Control"
+metaDescription: "Overall technologic details that might impact the experience"
+freshnessValidatedDate: never
+---
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+## Overview
+
+**Agent Control** is New Relic's solution for managing infrastructure-level instrumentation at scale. It addresses the high toil and complexity of deploying, configuring, and updating agents across New Relic and OpenTelemetry ecosystems. By acting as a single, lightweight agent supervisor, Agent Control automates these tasks to provide a seamless and consistent experience across your fleets.
+
+## The challenge of manage observability at scale
+
+Significant challenges rise when managing observability infrastructure at scale. Manually deploying and configuring multiple agents across diverse environments — like Kubernetes clusters and on-host systems — is time-consuming, prone to errors, and difficult to keep consistent. This manual toil makes it hard to quickly add new instrumentation, optimize configurations, and stay up to date with the latest security patches and features.
+
+Agent Control provides a single, unified solution for managing all your New Relic and OpenTelemetry agents. It runs as a lightweight supervisor on your hosts or within your Kubernetes clusters, communicating with the New Relic platform to receive remote configurations and deployment instructions. This eliminates the need for manual intervention on each host, allowing you to manage your entire fleet from a single, centralized location.
+
+## How Agent Control Works
+
+
+The image below shows how Agent Control connects Agent Management with Fleet Control:
+
+   <img
+     title="Agent Control architecture"
+     alt="Agent Control architecture for Agent Management"
+     src="/images/agent-control-architecture.png"
+   />
+
+
+Agent Control's power comes from its use of a declarative, GitOps-like approach to agent management. It treats the desired state of your instrumentation as a single source of truth, ensuring that what you configure in New Relic Control is what's running on your hosts. This process is driven by key underlying technologies.
+
+### GitOps-like Principles with Flux
+Agent Control leverages Flux, a powerful and widely adopted tool, to manage the state of your agents. This means you define your desired configuration once, and Agent Control automatically ensures that your agents conform to this state. This declarative approach provides several key benefits:
```

Comment:
Would it be OK to include a link to https://fluxcd.io/?

Reply:
Since we have a single architecture page, maybe we need to specify that this Flux tool is used in K8s.